man/systemd.exec.xml

```diff
@@ -759,10 +759,11 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         <term><varname>CPUAffinity=</varname></term>
 
         <listitem><para>Controls the CPU affinity of the executed processes. Takes a list of CPU indices or ranges
-        separated by either whitespace or commas. CPU ranges are specified by the lower and upper CPU indices separated
-        by a dash. This option may be specified more than once, in which case the specified CPU affinity masks are
-        merged. If the empty string is assigned, the mask is reset, all assignments prior to this will have no
-        effect. See
+        separated by either whitespace or commas. Alternatively, takes a special "NUMA" value in which case systemd
```

Comment:
all our textual identifiers we use on the right-hand side of settings are lowercase afaict, this one should do. i.e. make "numa" the special word here, not "NUMA"